web/src/pages/salesforce/index.js

```diff
@@ -0,0 +1,263 @@
+import "./salesforce.scss";
+
+const SalesforcePage = ({ data }) => {
+    
+    return (
+        <Layout pageTitle="Salesforce | Awesome Inc">
+            
+            {/* Header section */}
+            <section id="header">
+                <Container fluid>
+                    <Row className="header-section">
+                        <Col className="header-text">
+                            <Title className="text-uppercase">Salesforce career accelerator</Title>
+                            <BrandButton className="my-3">START YOUR APPLICATION</BrandButton>
+                            <p>Get started now in 30 seconds or less</p>
+                        </Col>
+                    </Row>
+                <Col xs={{ span: 5, offset: 3 }} className="text-center fw-bold fs-5 lh-lg mb-5">
+                    This part time course will put you on the path towards a
+                    promising career in the Salesforce ecosystem. Work
+                    together with our team and peers to earn your first
+                    Salesforce Certification and learn key tech career skills.
+                </Col>  
+                </Container>
+            </section>
+
+            {/* Upcoming Program Dates Section*/}
+            <section id="upcoming-program-dates">
+                <Container fluid className="background--grey pt-4">
+                    <Row>
+                        <h1 className="title">UPCOMING PROGRAM DATES</h1>
+                    </Row>
+                    <Row className="dates">
+                        <Col className="winter-dates">
+                            <h4 className="fw-bold">WINTER 2023</h4>
+                            <h4 className="fw-bold">2.15.23</h4>
+                            <h4>TO</h4>
+                            <h4 className="fw-bold">05.15.23</h4>
+                        </Col>
+                        <Col className="spring-dates">
+                            <h4 className="fw-bold">SPRING 2023</h4>
+                            <h4 className="fw-bold">03.15.23</h4>
+                            <h4>TO</h4>
+                            <h4 className="fw-bold">06.15.23</h4>
+                        </Col>
+                        <div className="button">
+                            <BrandButton>LET'S CHAT</BrandButton>
+                        </div>
+                    </Row>
+                </Container>
+            </section>
+
+            {/* Why Salesforce Section*/}
+            <section id="why-salesforce">
+                <Container>
+                    <Col className="title">                   
+                        <Title className="text-uppercase">Why learn salesforce?</Title>
+                    </Col>
+                    <Col className="paragraph">
+                        <p>
+                            Career opportunities in the Salesforce ecosystem are growing rapidly, and offer a low hurdle to entry-level <br />
```

Comment:
Looks like we still have the `<br>` tags. We shouldn't need those. You can do it similarly to how the `<Col>` on line 29 is. That will wrap the paragraph's instead of manually using `<br>`